exams/types.js

```diff
@@ -0,0 +1,16 @@
+types = function (array) {

```

Comment:
prefer arrow function 
```suggestion
const types = (array) => {
```